tests/foreman/installer/test_installer.py

```diff
@@ -1869,5 +1880,39 @@ def test_satellite_installation_with_options(sat_ready_rhel):
     assert len(result.stdout) == 0
     result = sat_ready_rhel.cli.Health.check()
     assert 'FAIL' not in result.stdout
+    # Verify foreman-rails-cache-store option works
     assert sat_ready_rhel.execute('rpm -q foreman-redis').status == 0
     assert sat_ready_rhel.execute('grep "type: redis" /etc/foreman/settings.yaml').status == 0
+    # Verify foreman-proxy-content-pulpcore-hide-guarded-distributions option works
+    sat_ready_rhel.execute(
+        'firewall-cmd --permanent --add-service RH-Satellite-6 && '
+        'firewall-cmd --add-service RH-Satellite-6'
```

Comment:
I would prefer `firewall-cmd --permanent --add-service <service> && firewall-cmd --reload`(add permanent rules without applying, then reloads and applies) or maybe `firewall-cmd --add-service <service> && firewall-cmd --runtime-to-permanent` (applies rule, then saves it permanently)

Reply:
@rmynar updated the pr :)